### DISCUSSION TITLE
Un punto en la práctica 3 de CSS

### DISCUSSION THREAD

**Discussion by IsaiVelasco** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#265] [General]
Disculpen, una pregunta: ¿A qué se refiere con "Planeación de campaña de vacunación )Un poco de mercadotecnia para llegar al sitio)"? ¿Vamos a hacer como publicaciones para Redes sociales o algo así además del maquetado de la imagen?

**Comment by Ingrzs** [comment]
Yo entendí esto, que tendrías que poner texto/contenido en la página para animar a las personas a vacunarse ya que el objetivo de la pagina es vacunar a la población.

Te habras dado cuenta que la pagina que puso el mc esta en inglés pues tendrias que poner lo tuyo propio.

El texto/contenido o campaña debe ser llamativo ....ya que en si cuando un cliente visita tu página este debe consumirlo según tu nicho. (Lo que seria una landin page)
Si no busca en google como es una landin page y veras cual es el objetivo.

Igual puedes ir adelantando replicando la página (tiene que quedar casi igual) y si no pues después veremos si el mc nos dice que la campaña sería un wireframe o buyer o diseñar la promoción de la campaña

**Reply by IsaiVelasco** [reply]
Si. Lo que pasa es que dice mercadotecnia para llegar al sitio, no mientras se esté en el sitio. Ahí me confundí. Gracias por tu respuesta.

**Comment by GloriaCoralCerecedo** [comment]
Hola, buenas noches, disculpen lo que entendí es que se va a hacer mercadotecnia del sitio me imagino que como anuncio o cosas, ¿No? Cómo mencionaron

Pero ... Igual se va a maquetar y hacer el diseño lo más parecido a cómo está en la imagen (?)

Y claro el bonus es que pueda tener más pestañas o estilos

**Reply by JuanSalvador73** [reply]
Por mercadoctecnia se refieren a la publicidad, slogan, frases que ese landing page va a contener para llamar la atención de público y lo invite a vacunarse.

Y aunque la idea es hacerlo semejante a la imagen mostrada, uno como desarrollador claro que le tiene que dar su toque personal, respetando los lineamientos, claro.

**Reply by GloriaCoralCerecedo** [reply]
Oh vale gracias
